src/handlers/web3.js

```diff
@@ -433,23 +433,25 @@ export const getDataForNftTransfer = (from, to, asset) => {
 };
 
 /**
- * @desc estimate gas limit
- * @param {Object} [{selected, address, recipient, amount, gasPrice}]
+ * @desc build transaction object
+ * @param {Object} [{selected, address, recipient, amount, gasLimit}]
+ * @param {Provider} provider
+ * @param {String} network
  * @return {String}
  */
-export const estimateGasLimit = async (
-  { asset, address, recipient, amount },
-  addPadding = false,
-  provider = null,
-  network = NetworkTypes.mainnet
+
+export const buildTransaction = async (
+  { asset, address, recipient, amount, gasLimit },
```

Comment:
abc

Reply:
wanna turn on an eslint rule for this so we can forget about it?